_posts/2023-09-23-virtual-threads-crud.adoc

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: 'Writing CRUD applications using virtual threads'
+date: 2023-09-23
+tags: virtual threads, reactive, crud, database
+synopsis: 'Describe how you can implement a CRUD / RESTFul application using virtual threads and Quarkus.'
+author: cescoffier
+---
+:imagesdir: /assets/images/posts/virtual-threads
+
+Last week, we published a video demonstrating the creation of a CRUD application using virtual threads in Quarkus. It's as simple as adding the `@RunOnVirtualThread` annotation on your HTTP resource (or your controller class if you use the Spring compatibility layer).
```

Comment:
last week? got link to hat video? video link below not working for me.

Reply:
Changed the insertion, it should work now.